src/executor/current_thread/mod.rs

```diff
@@ -147,9 +147,18 @@ pub struct TaskExecutor {
     _p: ::std::marker::PhantomData<Rc<()>>,
 }
 
-/// Returned by the `turn` function
+/// Returned by the `turn` function.
 #[derive(Debug)]
-pub struct Turn(());
+pub struct Turn {
+    polled: bool
+}
+
+impl Turn {
+    /// `true` if any futures were polled at all and `false` otherwise.
+    pub fn has_polled(&self) -> bool {
```

Comment:
My initial cut would have been to name this `did_poll`, that is probably being influenced by my time w/ cocoa. I'm being told I'm wrong, so if everyone likes `has_polled`, that is fine w/ me.

Reply:
Personally, I think `has_polled` is more grammatically correct than `did_poll`. I suppose this could also just be `polled` but I think starting with a word like "did" or "has" better indicates that it's a predicate...